diff --git a/init.h b/init.h
index dcb664f8..7fd4e15e 100644
--- a/init.h
+++ b/init.h
@@ -4016,7 +4016,7 @@ struct option_t MuttVars[] = {
   ** edited. This option points to the location of the private keys.
   ** (S/MIME only)
   */
-  { "smime_pkcs7_default_smime_type", 	DT_STR, R_NONE, {.p=&SmimePkcs7DefaultSmimeType}, {.p="signed"} },
+  { "smime_pkcs7_default_smime_type", 	DT_STR, R_NONE, {.p=&SmimePkcs7DefaultSmimeType}, {.p="enveloped"} },
   /*
   ** .pp
   ** The application/pkcs7-mime ``.p7m'' type can contain EnvelopedData
